splinterd/Cargo.toml

```diff
@@ -41,7 +41,7 @@ cylinder = { version = "0.2.2", features = ["key-load"] }
 diesel = { version = "1.0", features = ["postgres"], optional = true }
 health = { path = "../services/health", optional = true }
 log = "0.4"
-log4rs = "1"
+log4rs = {version = "1", features = ["threshold_filter"]}
```

Comment:
Lint - need whitespace after { and before }.

Reply:
Added